### DISCUSSION TITLE
Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document.

### DISCUSSION THREAD

**Discussion by GT-740** [community/community#70242] [GitHub Education]
### Hi everyone,

Unable to register in https://education.github.com/pack/join due to these error messages:
- Please correct your [GitHub billing information](https://github.com/settings/billing/payment_information) with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.
- Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document.
---
Same issue also occurs for all of these people, the ones who responded in their posts and potentially many more:
https://github.com/orgs/community/discussions/70043
https://github.com/orgs/community/discussions/68725
https://github.com/orgs/community/discussions/70008
https://github.com/orgs/community/discussions/69755

The problem is that it tells you to re-check the information you've entered. In reality, it wants you to enter an incorrectly translated entries instead of an officially transliterated ones as [this answer](https://github.com/orgs/community/discussions/68725#discussioncomment-7163466) states. After translating first and last names through [bing](https://www.bing.com/translator) it actually accepted the form for me. I'd say it's a bug, it should work with any official variants written on the ID.

### A suggestion...(optional)

Compare to official first/last name variants instead of translated ones. Correct the error message and/or allow to open support ticket when such case is detected in future until the problem is fixed.

**Comment by YuuSanLearnCode** [comment]
thank you. That fix for me

**Comment by JaehunLee314** [comment]
This worked for me too. It seems like that even though I've uploaded English version of 
my school ID, the system somehow changed my name into Bing translation result. Why...

**Comment by Suzukitakeko** [comment]
Thanks!
